graylog2-web-interface/src/components/permissions/ValidationError.jsx

```diff
@@ -0,0 +1,44 @@
+// @flow strict
+import * as React from 'react';
+import styled, { type StyledComponent } from 'styled-components';
+
+import { Alert } from 'components/graylog';
+import { type ThemeInterface } from 'theme';
+import { type GranteesList } from 'logic/permissions/EntityShareState';
+import ValidationResult from 'logic/permissions/ValidationResult';
+
+const Container: StyledComponent<{}, ThemeInterface, Alert> = styled(Alert)`
+  margin-top: 20px;
+  max-height: 240px;
+  overflow: auto;
+`;
+
+const List = styled.ul`
+  list-style: initial;
+  padding-left: 20px;
+
+  ul {
+    list-style: circle;
+  }
+`;
+
+type Props = {
+  validationResult: ValidationResult,
+  availableGrantees: GranteesList,
+};
+
+const ValidationError = ({ validationResult, availableGrantees }: Props) => {
+  const pastOwners = validationResult.errorContext.selectedGranteeCapabilities.toJS().map(
+    (grn) => availableGrantees.find((grantee) => grantee.id === grn)?.title,
+  ).join(', ');
+
+  return (
+    <Container bsStyle="danger">
+      <List>
+        <li>{`Removing the following owners ${pastOwners} will leave the entity ownerless.`}</li>
```

Comment:
We could structure this message similar to the dependency warning: 
```
<li>
  Removing the following owners will leave the entity ownerless:</br>
  {pastOwners.map((owner, key) => (
    <span key={owner.id}>
      {_cap(owner.type)} <i>{owner.title}</i>
      {key !== pasOwners.size - 1 && ', '}
    </span>
  ))}
</li>
```
`pastOwners` is an array of Grantees in this case.
This way the message is still readable, even if there are many previous owners, the owner names can be identified easier, because of the `<i>` and it is consistent compared with the dependency warning.